lib/hanami/model/plugins/timestamps.rb

```diff
@@ -24,7 +24,7 @@ class InputWithTimestamp < WrappingInput
           def initialize(relation, input)
             super
             columns     = relation.columns.sort
-            @timestamps = (columns & TIMESTAMPS) == TIMESTAMPS
+            @timestamps = (columns & TIMESTAMPS).count.positive?
```

Comment:
isn't `!(columns & TIMESTAMPS).empty?` more efficient?

And just for the sake of knowledge let me ask you,  I know it was previously there but isn't better to have the boolean check (either `.count.positive?` or `.empty?` placed in `timestamp?` method instead of in the initializer?